nifi-nar-bundles/nifi-tcp-bundle/nifi-tcp-processors/src/main/java/org/apache/nifi/processors/gettcp/DisconnectListener.java

```diff
@@ -0,0 +1,6 @@
+package org.apache.nifi.processors.gettcp;
+
+public interface DisconnectListener {
+
+    void onDisconnect(ReceivingClient client);
+}
```

Comment:
This class is actually not used anywhere. Accidental leftover (my bad), so please remove